src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java

```diff
@@ -0,0 +1,54 @@
+package de.tum.in.www1.artemis.config;
+
+import java.util.List;
+
+import org.springframework.boot.actuate.health.*;
+import org.springframework.cloud.client.discovery.health.DiscoveryCompositeHealthContributor;
+import org.springframework.messaging.simp.user.SimpUserRegistry;
+import org.springframework.stereotype.Component;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+
+@Component
+public class MetricsBean {
+
+    public MetricsBean(MeterRegistry meterRegistry, SimpUserRegistry simpUserRegistry, List<HealthContributor> healthContributors) {
+        Gauge.builder("artemis.instance.websocket.users", simpUserRegistry, SimpUserRegistry::getUserCount).strongReference(true)
+                .description("Number of users connected to this Artemis instance").register(meterRegistry);
+
+        for (HealthContributor healthContributor : healthContributors) {
+            if (healthContributor instanceof HealthIndicator) {
+                HealthIndicator healthIndicator = (HealthIndicator) healthContributor;
+                Gauge.builder("artemis.health", healthIndicator, h -> mapHealthToDouble(h.health())).strongReference(true).description("Artemis Health Indicator")
```

Comment:
You could define constants for "artemis.health", "Artemis Health Indicator", "healt(h)indicator". But this could also hurt readability.